wordpress-mobile/WordPress-iOS
# Muriel Colors: Add new API and navigation bar changes

Refs #11683 

This sets up the main API for using Muriel colors in the apps. As an intial example, this build changes the colors of the navigation bar to the colors, behind a feature flag.

![muriel navbar mk2](https://user-images.githubusercontent.com/517257/59800696-22541500-929c-11e9-81e3-94dcae6b01a5.jpg)


To test:
- Read the API, see what you think. It's all strongly typed Swift.
- Turn the flag on/off and see how the navigation bar responds.

Update release notes:
- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`. (**No changes**)
